batch/pixetl.dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM globalforestwatch/pixetl:latest
```

Comment:
This will pull from dockerhub, which is fine. There is a webhook which always rebuilds the latest image when master branch is updated.
We also publish to AWS ECR via Terraform. We should check which implementation works better and then reference that one.